test/lmp/model/_base/conftest.py

```diff
@@ -0,0 +1,37 @@
+r"""Setup fixture for testing :py:mod:`lmp.model.BaseModel`."""
+
+from typing import Type
+
+import pytest
+import torch
+
+from lmp.model._base import BaseModel
+
+
+@pytest.fixture
+def subclss_model_clss() -> Type[BaseModel]:
+    r"""Simple ``BaseModel`` subclass."""
+    class SubclssModel(BaseModel):
+        r"""Only implement ``tknz`` and ``dtknz``."""
+
+        def forward(self, batch_prev_tkids: torch.Tensor) -> torch.Tensor:
+            return ''.join(batch_prev_tkids)
```

Comment:
Return dummy tensor with shape `(B, S, V)`.
For example: `torch.zeros(batch_prev_tkids.size(0), batch_prev_tkids.size(1), 100)`